src/test/java/es/us/isa/restest/testcases/objectfunction/DiversityTest.java

```diff
@@ -0,0 +1,75 @@
+package es.us.isa.restest.testcases.objectfunction;
+
+import es.us.isa.restest.testcases.TestCase;
+import es.us.isa.restest.util.TestManager;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
```

Comment:
Avoid using these libraries if possible. For checking doubles, it can be done with JUnit, e.g., `assertEquals(expected, actual, delta)`